### DISCUSSION TITLE
co-simulation with sumo

### DISCUSSION THREAD

**Discussion by aslirey** [carla-simulator/carla#4991] [Q&A]
Hello everyone,

I will generate a senario of co-simulation with sumo. When using run_synchronization.py, is it possible to add sensor to a vehicle? Thanks in advance.

**Comment by Aim-ez** [comment]
Hello, I know this was asked a while ago, but I am hoping to attach a CARLA sensor to a SUMO spawned vehicle in a co-simulation, and I was wondering if you found a solution to this? Thanks!
